drivers/src/printer.rs

```diff
@@ -25,7 +25,7 @@ impl uWrite for MutablePrinter {
     #[inline(never)]
     fn write_str(&mut self, _str: &str) -> Result<(), Self::Error> {
         #[cfg(feature = "emu")]
-        caliptra_drivers::Uart::default().write(_str);
```

Comment:
nit : Instead of MutablePrinter, let's call it just Printer.

Reply:
That's the name it had before; I didn't change it.

Reply:
I know. I regret naming it like that.

Reply:
Renamed.